### PR TITLE
Fixed propagation of request body sample.

### PR DESCRIPTION
**Pull request by azaslonov**

### CODE REVIEW

File: src/models/response.ts [hunk 1]
```diff
@@ -15,11 +15,11 @@ export class Response {
      * Returns "true" if this response is meaningful from documentation prospective.
      */
     public isMeaningful(): boolean {
-        return !!this.description || this.representations.some(x => !!x.typeName || !!x.example);
+        return !!this.description || this.representations.some(x => !!x.typeName || x.examples.length > 0);
```
Comment (ygrik) [inline src/models/response.ts:18]: do we need to be aware if there are no examples? - x.examples?

File: src/models/response.ts [hunk 2]
```diff
@@ -15,11 +15,11 @@ export class Response {
...
     }
 
     public meaningfulRepresentations(): Representation[] {
-        return this.representations.filter(x => !!x.typeName || !!x.example);
+        return this.representations.filter(x => !!x.typeName || x.examples.length > 0);
```
Comment (ygrik) [inline src/models/response.ts:22]: the same for **x.examples**

File: src/components/operations/operation-details/ko/runtime/operation-details.ts [hunk 3]
```diff
@@ -318,8 +318,8 @@ export class OperationDetails {
             definition.name = representation.typeName;
         }
 
-        if (representation.example) {
-            definition.example = representation.example;
+        if (representation.examples.length > 0) {
```
Comment (ygrik) [inline src/components/operations/operation-details/ko/runtime/operation-details.ts:321]: examples can be undefined

File: src/models/console/consoleRepresentation.ts [hunk 4]
```diff
@@ -6,10 +6,10 @@ export class ConsoleRepresentation {
     public readonly schemaId?: string;
     public readonly typeName?: string;
 
-    constructor(contract: Representation) {
-        this.sample = contract.example;
-        this.contentType = contract.contentType;
-        this.schemaId = contract.schemaId;
-        this.typeName = contract.typeName;
+    constructor(representation: Representation) {
+        this.sample = representation.examples.length > 0 ? representation.examples[0].value : "";
```
Comment (ygrik) [inline src/models/console/consoleRepresentation.ts:10]: examples can be undefined

**Review by ygrik — APPROVED** [review]

File: src/models/response.ts (hunk 1 above)
Comment (azaslonov) [reply src/models/response.ts]: Yep, added checks.

**Pull request merged**